src/Parser/BalanceDruid/Modules/Spells/UnempoweredLs.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatNumber } from 'common/format';
+
+class UnempoweredLS extends Module {
+  _castQueue = {
+      Empowered: false,
+      Enemies: 0,
+    };
+  casts = [];
+  _lunarEmpsOn = false;
+
+  suboptUmempLS = 0;
+
+  isLunarStrike(event) {
+    const spellId = event.ability.guid;
+    return spellId === SPELLS.LUNAR_STRIKE.id;
+  }
+  isLunarEmpowerment(event){
+      const spellId = event.ability.guid;
+      return spellId === SPELLS.LUNAR_EMP_BUFF.id;
+  }
+
+  on_byPlayer_cast(event) {
+    if (this.isLunarStrike(event)) {
```

Comment:
To prevent a lot of indentation I recommend inverting this check and returning an empty value if it fails.